src/Microsoft.AspNet.SignalR.Core/Infrastructure/MemoryPoolTextWriter.cs

```diff
@@ -3,10 +3,11 @@
 using System;
 using System.Text;
 using System.IO;
+using System.Globalization;
 
 namespace Microsoft.AspNet.SignalR.Infrastructure
 {
-    public class MemoryPoolTextWriter : TextWriter
+    internal class MemoryPoolTextWriter : TextWriter
```

Comment:
@davidfowl I made some types internal - was there a specific reason that they were public before (e.g. did we expect people to derive from them and overwrite some functionality)